src/main/java/bio/terra/testrunner/common/HelmRelease.java

```diff
@@ -0,0 +1,15 @@
+package bio.terra.testrunner.common;
+
+import java.util.Map;
+
+public class HelmRelease {
```

Comment:
nit: could you add a description here? I think this is going to have one version per service right? if so, that information, or whatever you expect the map keys to be would be helpful.

Reply:
Added description to `HelmRelease`.